docs/machine-learning/tutorials/object-detection-custom-vision-onnx.md

```diff
@@ -0,0 +1,388 @@
+* Azure account. If you don't have one, [create a free Azure account](https://aka.ms/AMLFree).
+
+## Select the right machine learning task
+
+## Create the Model
+
+### Create the Custom Vision Project
+
+Log into the [Microsoft Custom Vision service](https://www.customvision.ai/) and select "New Project".
+
+In the "New Project" dialog, fill out the following required items:
+
+- Set the "Name" of the Custom Vision project as **StopSignDetection**.
+- Select the "Resource" you will use. This is an Azure resource that will be created for the Custom Vision project. If none is listed, one can be created by selecting the **Create new** link.
+- Set the "Project type" as **Object Detection**.
+- Set the "Classification Types" as **Multiclass** since there will be one class per image.
+- Set the "Domain" as **General (compact) [S1]**. The compact domain will allow you to download the ONNX model.
+- For "Export capabilities" select **Basic platforms** to allow the export of the ONNX model.
+
+Once the above fields are filled out click the **Create project** button.
+
+### Add images
+
+With the project created, click on the **Add images** button to start adding images for the model to train on. Select the stop sign images that was downloaded in the file browser.
+
+Select the first image that is shown. You will be able to select objects that are in the image that you want the model to detect. Select the stop sign in the image. A popup will display and set the tag as **stop-sign**. Do this for all of the remaining images. Some images will have more than one stop sign in it so be sure to mark all that are in the images.
+
+### Train the model
+
+With the images uploaded and tagged the model can now be trained. Click on the **Train** button.
+
+A popup will display asking what type of training to use. Select **Quick training** and click the **Train** button.
+
+### Download the ONNX model
+
+Once training is completed click on the "Export" button. When the popup displays click on the "ONNX" selection to download the ONNX model.
+
+### Analyze ONNX model
+
+Unzip the ONNX file since it downloads as a zip file. The folder will contain several files, but the two that we will use in this tutorial are the following:
+
+- **labels.txt** is a text file containing the labels that were defined in the Custom Vision service.
+- **model.onnx** is the ONNX model that we will use to make predictions in ML.NET.
+
+In order to build our ML.NET pipeline we will need the names of the input and output column names. To get this we can use Netron, a [web](https://netron.app/) and [desktop](https://github.com/lutzroeder/netron/releases/) app that can analyze ONNX models and show its architecture.
+
+1. When using either the web or desktop app of Netron, open the ONNX model in the app. Once it opens it will display a graph. This graph will tell you a few things that you will need in order to build the ML.NET pipeline for predictions.
```

Comment:
Rather than showing the individual nodes, to simplify this section I think you can instruct them to:

1. Open the model in Netron
2. Click on the first node. This should display a pane with the Input / Output columns.

![image](https://user-images.githubusercontent.com/46974588/173267774-f7c3311f-2e88-42ef-8828-faea00bbe3ee.png)


Then you can go into the respective columns and what they mean.